README.md

```diff
@@ -121,13 +121,19 @@ If you decided to index, retrieve and convert to the TFRecord format, you first
 need to download and extract the TREC-CAR data:
 ```
 TRECCAR_DIR=./treccar/
+DATA_DIR=./data
+mkdir ${DATA_DIR}
+
 wget http://trec-car.cs.unh.edu/datareleases/v2.0/paragraphCorpus.v2.0.tar.xz -P ${TRECCAR_DIR}
 wget http://trec-car.cs.unh.edu/datareleases/v2.0/train.v2.0.tar.xz -P ${TRECCAR_DIR}
 wget http://trec-car.cs.unh.edu/datareleases/v2.0/benchmarkY1-test.v2.0.tar.xz -P ${TRECCAR_DIR}
+wget https://storage.googleapis.com/bert_models/2018_10_18/uncased_L-24_H-1024_A-16.zip -P ${DATA_DIR}
```

Comment:
This checkpoint should not be used because it was pretrained on TREC-CAR's test set. We should use this one instead, which was pretrained only on TREC-CAR's training set:
```suggestion
wget https://storage.googleapis.com/bert_treccar_data/pretrained_models/BERT_Large_pretrained_on_TREC_CAR_training_set_1M_iterations.tar.gz -P ${DATA_DIR}
```

Reply:
Thanks for the heads up!